KsApi/extensions/Optional+Numeric.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
```

Comment:
I think we should delete this file and its corresponding test file. I don't think it's necessary.

Reply:
I have an argument for this but it's still subjective. Let me give some example.

There are cases where we have expressions similar to 
`(someValueA ?? 0)` + `( someValueB ?? 0)`, with the new extension you can now have this instead
`someValueA.orZero` + `someValueB.orZero`

Another example is this
`let someNotNilValue: Int? = 3`
`let someNilValue: Int? = nil`

`let value = someNotNilValue + someNilValue` //  you can't do this without introducing a bracket.
so you'll have to do
`let value = someNotNilValue + (someNilValue ?? 0)`

but with the new extension you can do this instead.
`let value = someNotNilValue + someNilValue.orZero`

Makes the code a little easier to read, and also reduces using brackets.

Reply:
I think `orZero` makes the code less readable actually. The nil coalescing operator is a powerful part of the Swift language and I think when you see one it immediately suggests you are handling an optional with a default case. There will probably be a time in the future where we don't want to default to zero for a Float or Double so I think this extension has a very small, specific use case.